testing/engine/variables.go

```diff
@@ -85,7 +85,7 @@ SecRule ARGS_GET_NAMES "t1" "id:400,log"
 
 SecRule ARGS_NAMES "jsessionid" "id: 500, log, phase:2"
 SecRule ARGS_NAMES "pineapple" "id: 600, log, phase:2"
-SecRule ARGS "(?:^|[^\\])\\[cdeghijklmpqwxyz123456789]" "id:700,log,phase:2"
+SecRule ARGS "(?:^|[^\\\\])\\\\[cdeghijklmpqwxyz123456789]" "id:700,log,phase:2"
```

Comment:
Just want to double clarify since important - I'm not sure if this is correct until https://github.com/coreruleset/coreruleset/issues/2778

If this isn't correct though it means are neither raw nor quoted but a hybrid, and we'd need a custom unquoter

Reply:
We deprecated this `\` escaping since some time ago. Now CRS uses `\x5c`, which is portable between implementations (hopefully for Coraza too).

Reply:
Thanks, switched to that